### PR TITLE
Change menu cancel buttons to back

### LINKED ISSUE #15398: The 'cancel' button should be renamed 'back' for the replay & map chooser/editor sections
In TD, RA, & D2K the map chooser/editor sections have a 'cancel' button to go back. This should be renamed 'back' as it's confusing to click 'cancel' after deleting maps. This is also the case for the replays section for RA & D2K. (TD uses 'back' in replays)
edit:
Also 'save map' under the map editor; after saving a map it shouldn't be 'cancel' after saving it, it should be 'back'.

https://github.com/OpenRA/OpenRA/blob/669cf01d80baa2e83473c753a393fd1fb0192e57/mods/common/chrome/map-chooser.yaml#L145
https://github.com/OpenRA/OpenRA/blob/669cf01d80baa2e83473c753a393fd1fb0192e57/mods/cnc/chrome/mapchooser.yaml#L119
https://github.com/OpenRA/OpenRA/blob/669cf01d80baa2e83473c753a393fd1fb0192e57/mods/common/chrome/replaybrowser.yaml#L285
https://github.com/OpenRA/OpenRA/blob/669cf01d80baa2e83473c753a393fd1fb0192e57/mods/common/chrome/editor.yaml#L172
https://github.com/OpenRA/OpenRA/blob/669cf01d80baa2e83473c753a393fd1fb0192e57/mods/cnc/chrome/editor.yaml#L174

### PR DESCRIPTION
**Pull request by Smittytron**
Closes #15398 

The only change I'm unsure of is the save map button. It can be argued that save map is a prompt instead of part of the menu navigation, and thus makes more sense as 'cancel'. We're splitting hairs either way so I went ahead and pushed it as requested by @FrameLimiter 

If anyone has the same hang-up I can take the save map change out.

### CODE REVIEW

File: mods/cnc/chrome/editor.yaml [hunk 1]
```diff
@@ -170,7 +170,7 @@ Container@SAVE_MAP_PANEL:
 			Y: PARENT_BOTTOM - 35
 			Width: 140
 			Height: 35
-			Text: Cancel
```
Comment (chrisforbes) [inline mods/cnc/chrome/editor.yaml]: Nak to this one.

**Review by chrisforbes — DISMISSED** [review]
Actually, no, nak to this whole thing.

**Review by chrisforbes — CHANGES_REQUESTED** [review]
If you change the save map panel button back, 👍

**Comment by Smittytron** [comment]
Updated

**Review by chrisforbes — DISMISSED** [review]

**Review by pchote — APPROVED** [review]

**Comment by pchote** [comment]
https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/06460dbf2f8f1fd766554c600088da140c99d0c7

**Pull request merged**